_studio/shared/include/mfx_config.h

```diff
@@ -125,4 +125,16 @@
 
 #define CMAPIUPDATE
 
+/*
+#if !defined(ANDROID)
+    #define MFX_ENABLE_HEVCEHW_REFACTORING
```

Comment:
@eistarov : what are your expectations to merge in this change? I am worry that we did not yet branch out for Q4 release this year.

Reply:
@dvrogozh the plan was to switch to refactored code-pass per-platform as we aligning pass-rates for corresponding validations. Nearest planned ETA for Linux is SKL switch on ww50. The purpose of this exact pull-request is to start collect your feedback/address concerns, it is ok to delay merge by some release means but I want at least to start review now. When you expect to branch out Q4 ?

Reply:
> When you expect to branch out Q4 ?

@eistarov : that's a question to @onabiull. I guess we want to merge in some TGL code and pass at least one validation circle.

BTW, I recommend to consider merging big changes right after branch out/release to allow full quarter for tests... Please, address CI concerns at the first place - build still fails for clang and gcc-9.2.